### PR TITLE
zh-trans: /boilerplates/verify-crds.md

### PR DESCRIPTION
**Pull request by idefav**
ref:https://github.com/servicemesher/istio-official-translation/issues/1115

### CODE REVIEW

File: content/zh/boilerplates/verify-crds.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Wait for all Istio CRDs to be created:
+等待所有的 Istio CRDs 创建完成：
```
Comment (rootsongjc) [inline content/zh/boilerplates/verify-crds.md]: CRDs -> CRD

**Comment by tanjunchen** [comment]
@rootsongjc  OK to approve

File: content/zh/boilerplates/verify-crds.md (hunk 1 above)
Comment (tanjunchen) [reply content/zh/boilerplates/verify-crds.md]: @idefav 	update file according to review

Comment (idefav) [reply content/zh/boilerplates/verify-crds.md]: updated

**Pull request merged**